packages/astroport/src/pair.rs

```diff
@@ -6,6 +6,8 @@ use crate::asset::{Asset, AssetInfo};
 use cosmwasm_std::{Addr, Binary, Decimal, Uint128};
 use cw20::Cw20ReceiveMsg;
 
+pub const DEFAULT_SLIPPAGE: &str = "0.5";
```

Comment:
Let's use u64 and then Decimal::percent() similar to what @romanovdavid recommended in previous PR

Reply:
Also default slippage should be 0.005 (0.5%)
We also need one more variable for max_slippage which would be 50%, and we shouldn't let user to set bigger than that

Reply:
As i know we can't use Decimal::percent() function for percent less than one